### DISCUSSION TITLE
Plans to add a HTTP Client implementation?

### DISCUSSION THREAD

**Discussion by timsuchanek** [uNetworking/uWebSockets.js#311] [Q&A] [answered]
It seems that if the server part is already very fast, exposing a client would also make sense.

Are there any plans to expose an HTTP client that can be used from Node.js?

Thanks!

**Comment by rizrmd** [comment]
Is this feature still planned ?

**Reply by jimellert** [reply]
I too would love to see a client capability.

**Comment by uNetworkingAB** [comment, marked as answer]
It's not, not planned. But it's not planned.

**Reply by jimellert** [reply]
Understood!
